pkg/registry/file/dynamicpathdetector/tests/profile_test.go

```
@@ -0,0 +1,167 @@
+	runtime.ReadMemStats(&before)
+
+	for i := 0; i < *profileIters; i++ {
+		if _, err := analyzer.AnalyzePath(paths[i%len(paths)], identifier); err != nil {
+			pprof.StopCPUProfile()
+			cpuFile.Close()
+			t.Fatalf("AnalyzePath iter %d: %v", i, err)
+		}
+	}
+
+	pprof.StopCPUProfile()
+	cpuFile.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `cpuFile.Close` is not checked

(errcheck)